share/wake/html/timeline_main.js

```diff
@@ -312,11 +312,6 @@ timeline.on('click', function (properties) {
     }
     let job = jobMap.get(parseInt(properties.item)).job;
     document.getElementById("job").innerHTML = job.job
-    if (job.usage.charAt(8) === '0') {
-        document.getElementById("job").style.color = "green";
-    } else {
-        document.getElementById("job").style.color = "red";
-    }
```

Comment:
Its unclear to me if anyone is depending on the coloring here but its not super color blind friendly and removing it simplifies the diff for this PR